assets/scss/modules/_tabbed-pane.scss

```diff
@@ -0,0 +1,61 @@
+.tab-content {
+  .highlight {
```

Comment:
When parent element doesn't have styles of its own, don't nest, just use single selector line (`.tab-content .highlight { ... }` )